docs/theia.md

```diff
@@ -0,0 +1,68 @@
+# Theia
+
+Theia is the command-line tool which provides access to Theia network flow
+visibility capabilities.
+
+## Table of Contents
+
+<!-- toc -->
+- [Installation](#installation)
+- [Compile](#compile)
+- [Usage](#usage)
+<!-- /toc -->
+
+## Installation
+
+Starting with Theia release v0.1, we publish the theia binaries for
+different OS / CPU Architecture combinations. Head to the [releases
+page](https://github.com/antrea-io/theia/releases) and download the
+appropriate one for your machine. For example:
+
+On Mac & Linux:
+
+```bash
+curl -Lo ./theia "https://github.com/antrea-io/theia/releases/download/<TAG>/theia-$(uname)-x86_64"
+chmod +x ./theia
+mv ./theia /some-dir-in-your-PATH/theia
+theia help
+```
+
+For Linux, we also publish binaries for Arm-based systems.
+
+On Windows, using PowerShell:
+
+```powershell
+Invoke-WebRequest -Uri https://github.com/antrea-io/theia/releases/download/<TAG>/theia-windows-x86_64.exe -Outfile theia.exe
+Move-Item .\theia.exe c:\some-dir-in-your-PATH\theia.exe
+theia help
+```
+
+## Compile
+
+If you made some changes on theia code and would like to compile
+and test locally, please run these commands inside the Theia main directory:
```

Comment:
this paragraph seems redundant. Maybe just

In order to compile `theia` the following commands should be run from Theia's main directory:

Reply:
Thanks, addressed.